src/main/java/org/fungover/storm/client/ClientHandler.java

```diff
@@ -43,9 +42,8 @@ public void run() {
             out.close();
             clientSocket.close();
         } catch (IOException e) {
-            statusCode = new HttpResponseStatusCodes();
             if (e.getMessage().contains("500"))
-                LOGGER.error(statusCode.getError500());
+                LOGGER.error("500 Internal Server Error\r\n");
```

Comment:
Instead of hard coding this can we use ResponseCode.INTERNAL_SERVER_ERROR.getCode()?

Reply:
I fixed it. 
Thanks